external-docs/docs/guides/errors/build-command-errors/multiple-observations-columns.md

```diff
@@ -1,4 +1,8 @@
-# Error - multiple observations columns defined
+# Error - multiple observations columns defined (Deprecated)
+
+## Deprecation warning
+
+This validation error will no longer be raised from csvcubed v0.1.18. This is because from v0.1.18, csvcubed supports pivoted multi-measure data sets which have more than one observations columns defined.
```

Comment:
Could you tell them explicitly to upgrade the version of csvcubed they have installed - ideally giving them the conda/pip commands necessary.